test/e2e-secondary-network/infra/prepare_sriov.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+pushd $THIS_DIR
+
+_usage="Usage: $0 [--setup] [--cleanup]
+                  [--sriov-interface <interface name>] [--sriov-vf-count <count>] [--host-name <name>]
+
+Configure/deconfigure SR-IOV VFs and deploy the SRIOV device plugin at the remove single node cluster.
```

Comment:
Would you change the second SRIOV to SR-IOV too?

Reply:
Probably let me fix it together with several other typos.

Reply:
took care of all the instances of SRIOV replaced to SR-IOV. There is some function names and "sriov" as a network name remains unchanged.